packages/@sanity/cli/test/__fixtures__/v3/working-typegen.json

```diff
@@ -1,3 +1,4 @@
 {
-  "schema": "./working-schema.json"
+  "schema": "./working-schema.json",
+  "generates": "./out/types.ts"
```

Comment:
we probably should have a test with a directory and one without a directory, but otherwise, 👍